model/lifecycle.go

```diff
@@ -580,7 +580,7 @@ func CreateBuildFromVersion(args BuildCreateArgs) (string, error) {
 	start = time.Now()
 
 	if err = tasksForBuild.InsertUnordered(args.Session); err != nil {
-		return "", errors.Wrapf(err, "error inserting task for build '%s'", buildId)
+		return "", err
```

Comment:
I would continue to wrap this and then use errors.Cause to unwrap it later if needed.

Reply:
done